poker-pals/src/routes.js

```diff
@@ -4,6 +4,7 @@ import profileComponent from './components/Profile.vue';
 import reportsComponent from './components/Reports.vue';
 import tablesComponent from './components/Tables.vue';
 import exampleComponent from './components/HelloWorld.vue';
+import bannedComponent from './components/Banned.vue';
 
 export default [
```

Comment:
How does browser's back button affect the history of navigation? Let's say you've logged in and navigated to the poker page. What happens when you click the back button? Can you join a game again?

`vue-router` may have a way of disabling the effect of the back button. Might be worth looking into for a true single page application behaviour.

Reply:
if you click the back button it sends you back to the page you were on before accessing poker-pals (your homepage for example). this is what i thought was required for single page navigation

Reply:
If you were on Login page and then authenticated to Tables page. Will clicking back take you to login? Cause it shouldn't.

Reply:
i wont it sends you back to google.com or what ever

Reply:
My main concern was with `props` since I'm guessing they will reset? That's why I asked if you are able to Log in > select a table > click back button > join a game again

Reply:
they will reset

Reply:
So, I'm guessing other pages will think you've disconnected? Would it work better if you stored the props as a local variable and used that to pass to other pages?